README.md

```diff
@@ -129,6 +129,66 @@ env "gorm" {
 }
 ```
 
+### Supported Features
+#### View
+> Note: The view feature is only available for logged-in users, run `atlas login` if you haven't already. To learn more about logged-in features for Atlas, visit [Feature Availability](https://atlasgo.io/features#database-features).
+
+To create a view, you can define the `ViewDef` method on a struct:
+```go
+type User struct {
+  gorm.Model
+  Name string
+  Age  int
+}
+
+// this is our view
+type WorkingAgedUsers struct {
+  Name string
+  Age  int
+}
+
+func (WorkingAgedUsers) ViewDef(driver string) []gormschema.ViewOption {
+  return []gormschema.ViewOption{
+    gormschema.BuildStmt(func(db *gorm.DB) *gorm.DB {
+      return db.Model(&User{}).Where("age BETWEEN 18 AND 65").Select("name, age")
+    }),
+  }
+}
+```
+Then load the view just like a regular model:
+
+```go
+stmts, err := gormschema.New("mysql").Load(&models.WorkingAgedUsers{}, &models.User{})
+```
+If you need to create a complex view definition, you can utilize the `CreateStmt` option to specify a raw CREATE VIEW query, as shown below:
+```go
+type BotlTracker struct {
+  ID   uint
+  Name string
+}
+
+func (BotlTracker) ViewDef(driver string) []gormschema.ViewOption {
+  var stmt string
+  switch driver {
+  case "mysql":
+    stmt = "CREATE VIEW botl_trackers AS SELECT id, name FROM pets WHERE name LIKE ?"
+  }
+  return []gormschema.ViewOption{
+    gormschema.CreateStmt(stmt, "botl%"),
+  }
+}
```

Comment:
```suggestion
// WorkingAgedUsers is mapped to the VIEW definition below.
type WorkingAgedUsers struct {
	Name string
	Age  int
}

func (WorkingAgedUsers) ViewDef(dialect string) []gormschema.ViewOption {
	switch dialect {
	case "mysql":
		// ...
	case "sqlite":
		// ...
	case "postgres"
		return gormschema.CreateStmt("CREATE VIEW ..."),
	}
}
```

Reply:
Use tabs instead of spaces.

Reply:
@a8m I'm using tabs for my code, but don't you think spaces are better for markdown than tabs? Since tabs are extremely long-looking for markdown, and spaces make the appearance consistent with the IDE
![image](https://github.com/ariga/atlas-provider-gorm/assets/25551568/7c0a7350-4fb4-47f5-b19e-6d331a9864bc)

Reply:
I prefer tabs, because this is how Go code looks like, but since the README already has a mix of those, I'm fine with both options.